src/commonMain/kotlin/kotools/types/number/NonZeroInt.kt

```diff
@@ -64,6 +65,11 @@ public sealed interface NonZeroInt : AnyInt {
     override fun toString(): String
 }
 
+/** Returns the negative of this integer. */
+@ExperimentalNumberApi
+@SinceKotoolsTypes("4.2")
```

Comment:
Use the [`ExperimentalSinceKotoolsTypes`](https://github.com/kotools/types/blob/08afde83a30c2fb499cd6653dae4c37acdffa870/src/commonMain/kotlin/kotools/types/Annotations.kt#L17) annotation instead.